certified-connectors/OzyApprovals/apiDefinition.swagger.json

```diff
@@ -1 +1,1285 @@
+                }
+              }
+            }
+          }
+        },
+        "x-ms-trigger": "single",
+        "summary": "When a Ticket SLA is Breached",
+        "description": "Triggers when a Ticket SLA is Breached at a particular approval stage.",
+        "operationId": "OnSLABreached",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "target_url": {
+                  "type": "string",
+                  "description": "Target Url",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
+                },
+                "operation": {
+                  "type": "string",
+                  "description": "operation",
+                  "title": "Trigger Type",
+                  "default": "SLA_BREACHED",
+                  "x-ms-visibility": "internal"
+                },
+                "process_id": {
+                  "type": "integer",
+                  "description": "approval_process_id",
+                  "title": "Approval Process",
+                  "format": "string",
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetApprovalProcesses",
+                    "value-path": "id",
+                    "value-collection": "msg",
+                    "value-title": "name"
+                  }
+                },
+                "stage_id": {
+                  "type": "integer",
+                  "description": "",
+                  "title": "Approval Stage",
+                  "format": "string",
+                  "x-ms-dynamic-list": {
+                    "operationId": "GetProcessApprovalStages",
+                    "itemValuePath": "id",
+                    "itemTitlePath": "name",
+                    "parameters": {
+                      "process_id": {
+                        "parameterReference": "body/process_id"
+                      }
+                    }
+                  },
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetProcessApprovalStages",
+                    "value-path": "id",
+                    "value-collection": "msg",
+                    "value-title": "name",
+                    "parameters": {
+                      "process_id": {
+                        "parameter": "process_id"
+                      }
+                    }
+                  }
+                }
+              },
+              "required": [
+                "target_url",
+                "operation",
+                "process_id",
+                "stage_id"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/get_approval_processes": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
Please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.